### DISCUSSION TITLE
Are the "jslint", "global modules" and "$tw" comments essential for TiddlyWiki to work?

### DISCUSSION THREAD

**Discussion by nilslindemann** [TiddlyWiki/TiddlyWiki5#6209] [Q&A] [answered]
Hi people,

I wonder, are [these comments](https://github.com/Jermolene/TiddlyWiki5/blob/9caba544eb04f56ed772340a2d8767974b2617c9/boot/boot.js#L13-L14) required for TiddlyWiki to work? And if so, in which code location are they used?

I tried removing them from the boot.js and the wiki then still seems to work, so I guess they are used by editor tools. But let's ask to be sure.

I know that the `/*\ ... \*/` comments are needed by TiddlyWiki ([in this location](https://github.com/Jermolene/TiddlyWiki5/blob/9caba544eb04f56ed772340a2d8767974b2617c9/boot/boot.js#L2305)).

**Comment by pmario** [comment, marked as answer]
The comments are not needed. As you wrote, they are used by editor tools.

**Reply by nilslindemann** [reply]
Thank you

**Comment by linonetwo** [comment]
Not needed. And as js community evolves, more people are using eslint now.

And I think the IIFE is not needed too? As webpack or rollup can provide IIFE automatically. And I guess, since we are using `export.xxx`, then the js file is in a context that won't pollute other files.

**Reply by pmario** [reply]
See this discussion: https://github.com/Jermolene/TiddlyWiki5/issues/5368 It seems we are wasting time and memory

**Reply by nilslindemann** [reply]
Interesting link
